src/Xamarin.Android.Build.Tasks/Tasks/GetAppSettingsDirectory.cs

```diff
@@ -18,10 +18,10 @@ public class GetAppSettingsDirectory : AndroidTask
 
 		public override bool RunTask ()
 		{
-			// Windows: C:\Users\Jonathan\AppData\Local\Xamarin\Mono for Android
+			// Windows: C:\Users\Jonathan\AppData\Local\Xamarin\.NET for Android
 			var appdata_local = Environment.GetFolderPath (Environment.SpecialFolder.LocalApplicationData);
 
-			AppSettingsDirectory = Path.Combine (appdata_local, "Xamarin", "Mono for Android");
+			AppSettingsDirectory = Path.Combine (appdata_local, "Xamarin", ".NET for Android");
```

Comment:
We can't change this path so easily, as some IDE components might use it.

Undo the changes to this file.